perf_benchmark/benchmark_input_stream_translator_integration_test.cc

```diff
@@ -164,6 +164,11 @@ std::string ParseGrpcMessageToJsonMessage(ProtoType proto,
   return message;
 }
 
+// Convert both JSON strings JSON str to nlohmann::json objects and compare.
+void ExpectJsonStrEqual(absl::string_view actual, absl::string_view expected) {
+  EXPECT_EQ(nlohmann::json::parse(actual), nlohmann::json::parse(expected));
```

Comment:
This is not good.  If EXPECT_EQ fail,  the error message will show the line here.  All tests using this function should point to this line.    It will be harder to debug the failures.

Reply:
Ah that's true. I updated to use an explicit variable to hold the expected json string in each unit test.